main.go

```diff
@@ -73,6 +73,8 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	var stepSize = flag.Int("batch-chunk-size", 3, "batch-chunk-size is used to control at most how many subscriptions will be created concurrently")
```

Comment:
May I ask that if `batch-chunk-size` is the flag, does it mean we need pre-set it in our CSV as well?
https://github.com/IBM/operand-deployment-lifecycle-manager/blob/master/bundle/manifests/operand-deployment-lifecycle-manager.clusterserviceversion.yaml#L283

Reply:
It has a default value "3". It is not a necessary flag

Reply:
Oh I see, thanks!